orchagent/routeorch.cpp

```diff
@@ -2185,7 +2182,24 @@ bool RouteOrch::addRoutePost(const RouteBulkContext& ctx, const NextHopGroupKey
             if (ol_nextHops.getSize() > 1
                 && m_syncdNextHopGroups[ol_nextHops].ref_count == 0)
             {
-                m_bulkNhgReducedRefCnt.emplace(ol_nextHops, 0);
+                if (m_syncdNextHopGroups[ol_nextHops].ref_count == 0)
```

Comment:
This seems to be unrelated to your PR. Can you please revert?

Reply:
Oh yes, this is what I was talking about earlier. I forgot to commit, let me fix that